core/src/main/resources/jenkins/install/SetupWizard/wizardUi.js

```diff
@@ -0,0 +1,8 @@
+var targetDiv = document.querySelector('#target-div');
```

Comment:
Not a huge deal, but variables declared here are in global scope and will be available everywhere. So maybe try avoid declaring unnecessary ones.

Reply:
I have changed variable name to `defaultSiteId`. and later we'll update the default site id so that'll be `defaultUpdateSiteId`

`var defaultSiteId = document.querySelector('#default-site-id');`

Reply:
What I was saying is that you could inline some calls to avoid declaring extra variables.